docs/deploy/howto-deploy-helm.md

```diff
@@ -26,10 +26,10 @@ Kubernetes cluster:
 - For `0.2.0` version of the Chart: [Azure Industrial IoT Helm Chart v0.2.0](https://github.com/Azure/Industrial-IoT/blob/helm/0.2.0/deploy/helm/azure-industrial-iot/README.md)
 - For `0.3.1` version of the Chart: [Azure Industrial IoT Helm Chart v0.3.1](https://github.com/Azure/Industrial-IoT/blob/helm_0.3.1/deploy/helm/azure-industrial-iot/README.md)
 - For `0.3.2` version of the Chart: [Azure Industrial IoT Helm Chart v0.3.2](https://github.com/Azure/Industrial-IoT/blob/helm_0.3.2/deploy/helm/azure-industrial-iot/README.md)
-- For `0.4.0` version of the Chart: [Azure Industrial IoT Helm Chart v0.3.3](https://github.com/Azure/Industrial-IoT/blob/helm_0.4.0/deploy/helm/azure-industrial-iot/README.md)
+- For `0.4.0` version of the Chart: [Azure Industrial IoT Helm Chart v0.4.0](https://github.com/Azure/Industrial-IoT/blob/helm_0.4.0/deploy/helm/azure-industrial-iot/README.md)
```

Comment:
>0.4.0 [](http://example.com/codeflow?start=7&length=5)

0.4.1 ?

Reply:
Those are the links to existing tags that we have. The link was changed to point to `helm_0.4.0` tag, which is correct, but I missed changing the text that is displayed. So `0.4.0` is actually correct here.